changes/14464-add-kolide-tables

```diff
@@ -0,0 +1,29 @@
+- added tables to the fleetd extension: 
+  - airport_util
+  - app_icons
+  - apple_silicon_security_policy
```

Comment:
```suggestion
```

Let's cut this one. Kolide says it's not meant for normal users and I don't even know what it does. Do other folks? https://github.com/fleetdm/fleet/pull/14613/files#diff-db85db77a2b18c277b644c5d1569c6408fa88edf832ce2b6460e8870a5079d1eR27